<launch>
    <include file="$(find gazebo_race)/launch/race.launch"/>
    <node name="map_server" pkg="map_server" type="map_server" args="$(find gazebo_nav)/map/result.yaml" output="screen">
    <param name="frame_id" value="map" />
    </node>

    <include file="$(find gazebo_nav)/launch/config/amcl/amcl_omni.launch" > </include>

    <node pkg="move_base" type="move_base" respawn="false" name="real_base" output="screen">
        <param name="base_global_planner" value="global_planner/GlobalPlanner"/>
        <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>
        <remap from = "/move_base/cancel" to = "/real_base/cancel" />
        <remap from = "/move_base/feedback" to = "/real_base/feedback" />
        <remap from = "/move_base/goal" to = "/real_base/goal" />
        <remap from = "/move_base/result" to = "/real_base/result" />
        <remap from = "/move_base/status" to = "/real_base/status" />
        <remap from = "/move_base_simple/goal" to = "/real_base_simple/goal" />
        <rosparam file="$(find gazebo_nav)/launch/config/move_base/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find gazebo_nav)/launch/config/move_base/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find gazebo_nav)/launch/config/move_base/global_planner_params.yaml" command="load" />
        <rosparam file="$(find gazebo_nav)/launch/config/move_base/dwa_local_planner_params.yaml" command="load" />
        <rosparam file="$(find gazebo_nav)/launch/config/move_base/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find gazebo_nav)/launch/config/move_base/global_costmap_params.yaml" command="load" />
    </node>

    <node pkg="gazebo_nav" type="multipath_bridge.py" name="multipath_bridge" output="screen"/>

    <node name="rviz" pkg="rviz" type="rviz" required="true"
        args="-d $(find gazebo_nav)/launch/config/rviz/multipath.rviz" />
</launch>